content/services/hosting/hosting_details.rst

```diff
@@ -9,45 +9,99 @@ offload work from your machine and/or infrastructure team. This list is not
 exhaustive, but is intended to give a better idea of the types of services we
 provide.
 
+
+Managed vs. Unmanaged
+---------------------
+
+Most of our hosting falls into two categories, managed or unmanaged.
+
+Managed
+^^^^^^^
+
+For managed hosting, we will take care of your system(s) using our configuration management which ensures your systems
+are always up to date, stable and tested configurations, and automated. We currently use `Chef`_ to manage all of our
```

Comment:
The double "and" in this list throws me off a bit, perhaps saying "stable with tested configurations, and automated" flows a bit better?

```suggestion
are always up to date, stable with tested configurations, and automated. We currently use `Chef`_ to manage all of our
```